projects/plugins/crm/includes/ZeroBSCRM.AJAX.php

```diff
@@ -792,17 +794,16 @@ function ZeroBSCRM_get_quote_template() {
 				$quote_val = '[QUOTEVALUE]';
 			}
 			if ( empty( $quote_date ) ) {
-				$quote_date = date( 'd/m/Y', time() );
+				$quote_date = gmdate( 'Y-m-d' );
```

Comment:
Trying to figure out what exactly this changes...both of them seem to result in a blank selector when selecting a quote template.

It seems there are three places `$quote_date` can be set...seems ripe for cleanup, but perhaps out of scope.

Reply:
I think cleaning up when we review the date functionality generally would be good. In this case the change is probably confusing due to how I created this PR then removed changes to put in other PRs. This line becomes `wp_date( get_option( 'date_format' ) )` and I see you've commented over there so I'll follow up there.